packages/cms-cli/commands/init.js

```diff
@@ -49,11 +54,13 @@ function initializeConfigCommand(program) {
         TRACKING_STATUS.STARTED
       );
 
+      createEmptyConfigFile();
+      handleExit(deleteEmptyConfigFile);
```

Comment:
Moved this into `init` so that both are explicitly only part of the initialization flow.